api/roleAccess/dtos/to-role-assignment-dto.ts

```diff
@@ -129,6 +129,7 @@ export function toDenySARoleAssignmentBody(
         requestDate: allocateRoleData.requestCreated,
         reviewer: currentUserId,
         infoRequired: allocateRoleData.accessReason === 'Request more information',
+        infoRequiredComment: allocateRoleData.comment,
```

Comment:
I don't understand why we need infoRequiredComment?
Connor already has taken the decision to store the reason of the specific access request under 'specificAccessReason'.

Reply:
Because there are 2 reasons. (You can see this in screenshot attached to JIRA)
1. Reason for case access
2. Reason for rejection
If specificAccessReason used then value  will be overwritten with rejection reason

Reply:
David's correct Matei, no issue here